eve/main.yml

```diff
@@ -17,6 +17,9 @@ models:
       path: eve/workers/pod.yml
       images:
         aggressor: eve/workers/unit_and_feature_tests
+        warp10:
```

Comment:
I'm letting eve build the warp10 image as it needs to be published on a registry in order that pods in other nodes can pick up the image.